config-model/src/main/java/com/yahoo/vespa/model/admin/LogForwarder.java

```diff
@@ -86,7 +90,11 @@ public Optional<String> getPreShutdownCommand() {
         getConfig(builder);
         var cfg = new LogforwarderConfig(builder);
         var home = cfg.splunkHome();
-        return Optional.of("/usr/bin/env SPLUNK_HOME="+home+" "+home+"/bin/splunk stop");
+        if (Files.exists(Path.of(home))) {
```

Comment:
it's very brittle if we're checking if a directory exists **on the configserver where the model runs** to decide whether to set up splunk forwarding.  Maybe we need some zone-wide config or a feature flag instead?  checking if the directory exists on the actual node would make more sense to me.